src/AppInstallerRepositoryCore/Rest/Schema/1_0/Interface.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "Rest/Schema/1_0/Interface.h"
+#include "Rest/Schema/IRestClient.h"
+#include "Rest/HttpClientHelper.h"
+#include "cpprest/http_client.h"
+#include "cpprest/json.h"
+
+using namespace web;
+using namespace web::json;
+using namespace web::http;
+using namespace web::http::client;
+using namespace AppInstaller::Repository::Rest::Schema;
+
+namespace AppInstaller::Repository::Rest::Schema::V1_0
+{
+    // Endpoint constants
+    const std::string ManifestSearchPostEndpoint = "/api/manifestSearch?";

```

Comment:
Use `constexpr std::string_view` for holding string literals.  (or `constexpr std::wstring_view` for wide literals below)

Reply:
changed all to use string_view

---
In reply to: [583302491](https://github.com/microsoft/winget-cli/pull/765#discussion_r583302491) [](ancestors = 583302491)